File: bAdkOde/Prototypes/OldPrototypes/exptest.java
import java.io.*;

public class exptest
{
       public static void main(String[] args) throws IOException
       {
              int iv1 = 0;
              int iv2 = 0;
              float fv1 = 0;
              float fv2 = 0;
              double dv1 = 0;
              double dv2 = 0;

              BufferedReader stdin = new BufferedReader(new InputStreamReader(System.in));

              System.out.println("Enter iv1:");
              iv1 = Integer.parseInt(stdin.readLine());

              System.out.println("Enter iv2:");
              iv2 = Integer.parseInt(stdin.readLine());

              System.out.println("Enter fv1:");
              fv1 = new Float(stdin.readLine()).floatValue();

              System.out.println("Enter fv2:");
              fv2 = new Float(stdin.readLine()).floatValue();

              System.out.println("Enter dv1:");
              dv1 = new Double(stdin.readLine()).doubleValue();

              System.out.println("Enter dv2:");
              dv2 = new Double(stdin.readLine()).doubleValue();

              System.out.println(dv2 / dv1);
              System.out.println(dv2 / fv1);
              System.out.println(dv2 / iv1);

              System.out.println(fv2 / dv1);
              System.out.println(fv2 / fv1);
              System.out.println(fv2 / iv1);

              System.out.println(iv2 / dv1);
              System.out.println(iv2 / fv1); 
              System.out.println(iv2 / iv1);
      }
}
 

File: bAdkOde/Prototypes/OldPrototypes/exptest.cpp
#include <iostream.h>

void main()
{
     int iv1, iv2;
     float fv1, fv2;
     double dv1, dv2;

     cout<<"Enter iv1:";
     cin>>iv1;

     cout<<"Enter iv2:";
     cin>>iv2;

     cout<<"Enter fv1:";
     cin>>fv1;

     cout<<"Enter fv2:";
     cin>>fv2;

     cout<<"Enter dv1, dv2:";
     cin>>dv1>>dv2;

     cout<<iv2/iv1<<"\n";
     cout<<iv2/fv1<<"\n";
     cout<<iv2/dv1<<"\n";
     
     cout<<fv2/iv1<<"\n";
     cout<<fv2/fv1<<"\n";
     cout<<fv2/dv1<<"\n";

     cout<<dv2/iv1<<"\n";
     cout<<dv2/fv1<<"\n";
     cout<<dv2/dv1<<"\n";
}
  
